bika/lims/browser/aggregatedanalyses/aggregatedanalyses.py

```diff
@@ -58,15 +58,36 @@ def __init__(self, context, request, **kwargs):
             }
         self.review_states = [
             {'id': 'default',
-             'title':  _('Results pending'),
+             'title': _('Assignment pending'),
+             'transitions': [{'id': 'submit'},
+                             {'id': 'cancel'},
+                             # {'id': 'assign'}
+                             ],
+             'contentFilter': {
+                 'review_state': ['sample_received'],
+                 'cancellation_state': 'active', },
```

Comment:
Assignment pending refers to those analyses that hasn't been assigned to an analyst (through Worksheet creation) yet. This state is defined in [`worksheetanalysis_workflow`](https://github.com/senaite/senaite.core/blob/master/bika/lims/profiles/default/workflows/bika_worksheetanalysis_workflow/definition.xml) and the desired state for this filter should be `unassigned`. Note that an analysis could be in an `unassigned` state but at the same time `to_be_verified` (this case would happen if the analysis was never assigned to a worksheet and the results introduced through Analysis Requests' Manage Results view). Since is not possible to assign analyses to a worksheet that are in a state other than `sample_received` or `attachment_due`, please keep `review_state` filtering too:
```python
'worksheetanalysis_review_state': ['unassigned'],
'review_state': ['sample_received', 'attachment_due']
```